src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+@use '../_variables/variables' as *;
+
+.page {
+  font-family: $PrimaryFont;
```

Comment:
The variable '$PrimaryFont' should be written in lowercase with hyphens according to the code style rules. Consider renaming it to '$primary-font'.